### DISCUSSION TITLE
MPI error out of box

### DISCUSSION THREAD

**Discussion by bookreader** [su2code/SU2#2344] [General]
Hi SU2 friends,

I am new to SU2. 
I git cloned SU2 from git repository: https://github.com/su2code/SU2
I have also tried SU2 8.0.1

I did
./meson.py build
sudo ./ninja -C build install

When I run
SU2_CFD inv_NACA0012.cfg
in QuickStart folder, I got this error:

 *** An error occurred in MPI_Win_create
 *** reported by process [3211657217,0]
 *** on communicator MPI_COMM_WORLD
 *** MPI_ERR_WIN: invalid window
 *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
 ***    and potentially your MPI job)

Since I did not build with -Dcustom-mpi=true -Dextra-deps=mpich
I assume when I run SU2_CFD mpi will not be involved.
Why am I getting MPI related errors?

How can I run the QuickStart without getting these errors?

Thanks very much!

**Comment by pcarruscag** [comment]
When the code is compiled with mpi it tries to initialize it even when running on a single core.
Thus you can either disable mpi when you build SU2 (see the meson flags) or try to fix the issue by using mpich.

**Comment by TobiKattmann** [comment]
If you run `mpirun --mca osc ucx -n 1 SU2_CFD ...` you will most likely not get those errors. Regarding implications on your network performance, or anything really, I dont really know [here are a few info](https://docs.open-mpi.org/en/v5.0.x/release-notes/networks.html).
